migrate2rocky/migrate2rocky.sh

```diff
@@ -480,9 +481,12 @@ collect_system_info () {
             --noheadings) ||
             exit_message "Can't find EFI mount.  No EFI  boot detected."
         kname=$(lsblk -dno kname "$efi_mount")
+        # shellcheck disable=SC2178
```

Comment:
Also there may be situations where multiple efi partitions exist (like mdraid partitions as explained below), so an array should be defined.

I don't know how to deal with this shellcheck warning too, so let's ignore it too.

Reply:
It is an array, but in this particular spot only the first entry needs to be worked with.  In this particular case we can change the line to `efi_disk=($(lsblk ...))` and later, `if [[ ${efi_disk[0]} ]] ...` Functionally they are the same, but it will be a better way to silence the warning than ignoring it.

Reply:
Fixed in https://github.com/rocky-linux/rocky-tools/pull/146/commits/2eb170dd76c1b330e20e7b88b2f5908cd6dae8e5.